### PR TITLE
Change default branch to main

### PR DESCRIPTION
**Pull request by daviddavis**
[noissue]

### CODE REVIEW

File: template_config.yml [hunk 1]
```diff
@@ -31,7 +31,7 @@ plugin_caps: PULP_FILE
 plugin_caps_short: FILE
 plugin_dash: pulp-file
 plugin_dash_short: file
-plugin_default_branch: master
+plugin_default_branch: main
```
Comment (mdellweg) [inline template_config.yml:34]: Didn't the template_config.yml also get a git_ref stanza?

**Review by mdellweg — APPROVED** [review]

File: template_config.yml (hunk 1 above)
Comment (mdellweg) [reply template_config.yml:34]: Wait, it was probably not regenerated due to "no-change".

**Comment by mdellweg** [comment]
Please think about the nightly build in pulp-oci-images

**Comment by daviddavis** [comment]
Thank you for the reminder as I forgot.

https://github.com/pulp/pulp-oci-images/pull/113

**Pull request merged**